terraform/environments/data-platform/lambda.tf

```diff
@@ -310,7 +310,36 @@ module "get_schema_lambda" {
     AllowExecutionFromAPIGateway = {
       action     = "lambda:InvokeFunction"
       principal  = "apigateway.amazonaws.com"
-      source_arn = "arn:aws:execute-api:${local.region}:${local.account_id}:${aws_api_gateway_rest_api.data_platform.id}/*/${aws_api_gateway_method.get_schema_for_data_product_table_name.http_method}${aws_api_gateway_resource.schema_for_data_product_table_name.path}"
+      source_arn = "arn:aws:execute-api:${local.region}:${local.account_id}:${aws_api_gateway_rest_api.data_platform.id}/*/${aws_api_gateway_method.update_data_product.http_method}${aws_api_gateway_resource.schema_for_data_product_table_name.path}"
```

Comment:
Could this be changed back? must've changed it accidentally

Reply:
whoops, yes will revert